src/test/java/io/confluent/connect/elasticsearch/DataConverterTest.java

```diff
@@ -422,4 +424,19 @@ public void testDoNotInjectPayloadTimestampIfAlreadyExists() {
 
     assertEquals(timestamp, actualRecord.sourceAsMap().get(TIMESTAMP_FIELD));
   }
+
+  @Test
+  public void testDoNotAddExternalVersioningIfDataStream() {
```

Comment:
this would throw an exception without the changes right?

Reply:
Without the changes, the test would fail rather than throw an exception because the version type would be set to external. In this case, I don't think an exception would be thrown until the bulk request is sent.